benchmark/matmul_benchmark.py

```diff
@@ -0,0 +1,145 @@
+import argparse
+import time
+
```

Comment:
it's probably fine, but does this script maybe need some environment variables?

Why im asking: the next script declares num threads as environment variables, i was wondering if this one needs that too.

Reply:
I've deleted the unnecessary code and added some comments

Reply:
It's run on a GPU, so it should be fine without setting num threads